### PR TITLE
have query table list show search bar; fix InstancesTables filter

### PR DESCRIPTION
**Pull request by jadami10**
This is a `ui` `bugfix`

- adds the searchbox to the query tables list. every since pagination was added here, it's become a lot more challenging to scroll through tables when you have > 10 tables
- this fixes the `InstancesTables` page to only show the provided instances. right now it shows empty tables for all the other instances

the broker page (and controller/server/minion) before
image

after
image

### CODE REVIEW

**Comment by jadami10** [comment]
@joshigaurava are you able to review this?

File: pinot-controller/src/main/resources/app/components/Homepage/InstancesTables.tsx [hunk 1]
```diff
@@ -17,23 +17,30 @@
...
+        .map((key) => {
+          const value = _.get(instances, key, []);
+          return (
+            <InstanceTable
+              key={key}
+              name={`${key}s`}
+              instances={value}
+              clusterName={clusterName}
+            />
+          );
```
Comment (jayeshchoudhary) [inline pinot-controller/src/main/resources/app/components/Homepage/InstancesTables.tsx:39]: I think we don't need `.value` here, we are simply filtering instances that don't have key.

```js
{order
  .filter((key) => _.has(instances, key))
  .map((key) => {
    const value = _.get(instances, key, []);
    return (
      <InstanceTable
        key={key}
        name={`${key}s`}
        instances={value}
        clusterName={clusterName}
      />
    );
  })
}
```

**Review by jayeshchoudhary — APPROVED** [review]
minor comment otherwise lgtm.

File: pinot-controller/src/main/resources/app/components/Homepage/InstancesTables.tsx (hunk 1 above)
Comment (jadami10) [reply pinot-controller/src/main/resources/app/components/Homepage/InstancesTables.tsx]: good call. forgot array already has those methods

**Review by Jackie-Jiang — APPROVED** [review]

**Pull request merged**